buildscripts/semantic-convention/templates/SemanticAttributes.java.j2

```diff
@@ -75,7 +75,7 @@ public final class {{class}} {
   {%- for attribute in attributes | unique(attribute="fqn") %}
   {%- if attribute.is_enum %}
   {%- set class_name = attribute.fqn | to_camelcase(True) ~ "Values" %}
-  {%- set type = to_java_return_type(attribute.attr_type.enum_type) %}
+  {%- set type = to_java_return_type(attribute.attr_type.string_type) %}
   {%- if attribute.attr_type.custom_values %}
```

Comment:
we should get rid of this `if` check and keep the type assignment as it was before

Reply:
Right okay, I will make the changes